pkg/webhook/utils/utils.go

```diff
@@ -0,0 +1,64 @@
+package utils
```

Comment:
If the only place we use this code is in `dispatcher.go` we probably don't need a separate `utils` package, it can just go in `dispatcher.go`.

Reply:
fixed